db/schema.rb

```diff
@@ -23,6 +23,7 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "suid"
+    t.integer "school_type", limit: 2
```

Comment:
Why does the migration have limit: 1 and the schema limit: 2?

Reply:
That, honestly, is a good question, given that at no point was it ever 2.

I always set integers used as enums to limit 1 - it (supposedly) causes them to be stored as a one-byte data field, making them super smallish. I'm not sure how or why db:migrate decided to change this on me, let me see if I can dig up a strong answer to this.